src/webpack.js

```diff
@@ -53,32 +67,48 @@ const getDappMetaData = () => {
 };
 
 /**
- *  @desc Get data from webpack build bundle file and combine to metadata object
- *  @param {function} callback return data with format {result:'',error:''}
+ * @desc Watching and Streaming webpack build process
```

Comment:
first, remove the `ing`. But streaming a a build process makes no sense for me. You stream the build file ..